src/main/java/uk/gov/hmcts/reform/em/stitching/service/dto/BundleDTO.java

```diff
@@ -14,15 +18,18 @@
     private int version;
     private String description;
     private String purpose;
+    private Instant dateCreated;
```

Comment:
These fields should be implemented in `AbstractAuditingDTO`

Reply:
I'll have a look, thanks